bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -43,9 +43,21 @@
         mSavedStateHandler = savedStateHandler;
     }
 
+    void clear(@NonNull Object target) {
+        String uuid = mObjectUuidMap.get(target);
+        if (uuid == null) {
+            return;
+        }
+        clearDataFromDisk(uuid);
```

Comment:
Good question. So the targets are held as a weak reference in the `mObjectUuidMap` so clearing them out here manually rather than letting them get cleared out automatically probably won't make a huge difference. On the other hand, I also don't see the need to keep either the target or the UUID around if we're trying to manually clear them...it shouldn't change the functionality at all. I might try to add a `mObjectUuidMap.remove` just to be clear on what's going on.